mne/tests/test_utils.py

```diff
@@ -104,6 +104,29 @@ def test_object_size():
                     msg='%s < %s < %s:\n%s' % (lower, size, upper, obj))
 
 
+def test_get_inst_data():
```

Comment:
need to add something like `"""Test _get_inst_data"""` (it's nice for consistency when running the suite)